src/modification.cpp

```diff
@@ -0,0 +1,79 @@
+/** \brief This modification file calculates the percentage of packets lost
+*
+* This file defines a function filter2 which takes two files as parameters
+* 
+* The first parameter is the input file which it reads
+* The second parameter is the output file where it writes the modified output
+*
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#include <iostream>
+#include <string.h>
+#include <stdio.h>
+
+using namespace std;
+
+void filter2(char input_file[], char output_file[]){
+    FILE * fp;
+    FILE * fp_out;
+	
+	fp = fopen(input_file,"r");
+    fp_out = fopen(output_file,"w");
+	
+    fseek(fp, 0, SEEK_END);
+    long size = ftell(fp);
+    fseek(fp, 0, SEEK_SET);
+    char *pData = new char[size+1];
+    fread(pData, sizeof(char),size,fp);
+    fclose(fp);
+    
+    /**
+    * Parsing the read data from csv file
+    */
+    char* pch;
+    pch = strtok(pData, ",");
+    float counter = 0.0, counter1 = 0.0, counter2 = 0.0, pkt_loss = 0.0, pkt_loss_percnt = 0.0;
```

Comment:
The total number of characters have exceeded 80. Please reduce them.

Reply:
Done

Reply:
Pleas check indentation here

Reply:
It is done